app/src/main/java/com/kickstarter/viewmodels/LoggedInViewHolderViewModel.kt

```diff
@@ -112,8 +117,10 @@ interface LoggedInViewHolderViewModel {
                 .addToDisposable(disposables)
 
             Observable.just(
-                environment.featureFlagClient()
-                    ?.getBoolean(FlagKey.ANDROID_PLEDGED_PROJECTS_OVERVIEW) ?: false
+                featureFlagClient.getBoolean(FlagKey.ANDROID_PLEDGED_PROJECTS_OVERVIEW) &&
```

Comment:
Now PPO is tied to both Feature Flags, the remote config one (will restrict app versions), and the backend one (will add/remove unique userID's )